### ISSUE TITLE
False positive with Yamllint key-duplicates on Azure pipelines

### ISSUE THREAD

**Issue by ArthurBrum** [github/super-linter#3335]
Today I had a problem with lint detecting a key-duplicates in my yaml pipeline

While calling a templete, I pass several parameters like this:

```
- template: /path-to/file.yml 
    parameters:
      param1: "foo"
      ${{ if condition1 }}:
        param2: "value1"
      ${{ else }}:
        param2: "Fooccess"
      param3: "fooX"
      
      param4: "fooY"
      ${{ if condition2 }}:
        param5: "value3"
      ${{ elseif condition3 }}:
        param5: "Fooccess"
      ${{ else }}:
        param5: "Foocked :("
```

And as I have two lines written exactly **${{ else }}:** yamllint is detecting as duplicated key, but its not a key, its just an azure expression that may happen to be duplicated with not problem.. 
Since this is a relatively recent azure implementation ([started on september/2021](https://learn.microsoft.com/en-us/azure/devops/release-notes/2021/sprint-192-update#new-yaml-conditional-expressions)), it is probably not beeing handled correctly :(
So I'm not sure if this fix will be done here at super-linter or at yamllint, but I think I should warn people here anyway

I solved this by placing a `# yamllint disable-line` at line above, as this is a false positive

## Expected behavior

super-linter should not detect it as a fail

## Steps to Reproduce

Use code example above in any pipe then you should see error
(no local run yet to put here, will install and run to add later)

## Logs

`/tmp/lint/filesLint/android_ci.yml:127:15: [error] duplication of key "${{ else }}" in mapping (key-duplicates)`

## Additional context

No additional context

**Comment by ferrarimarco** [comment]
Hi @ArthurBrum . This looks like an issue with yamllint, rather than a super-linter one. You can try opening an issue there :)
